### PR TITLE
Features/enhanced readme

### PR DESCRIPTION
**Pull request by mayaalsamarrai**
Please review and leave your comments.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,70 @@
 # VirtoCommerce.Inventory
-VirtoCommerce.Inventory module represents a store inventory management system.
 
-![Inventory management UI](https://cloud.githubusercontent.com/assets/5801549/15571240/01a76534-233a-11e6-8cd1-1281c7de22f9.png)
+## Overview
 
-# Documentation
-User guide: <a href="https://virtocommerce.com/docs/vc2userguide/fulfillment" target="_blank">Fulfillment</a>
+Inventory management is a system of stock level controlling and fullfilment centers management.
```
Comment (mvktsk) [inline README.md]: typo **fulfillment**

File: README.md [hunk 2]
```diff
@@ -1,24 +1,70 @@
...
+The VC Inventory module is customizable and supports metafields.
+
+The Inventory module content can be both exported and imported using the VC Export/Import functionality.
+
+![Inventory module](docs/media/screen-inventory.png)
+
+Fields description :
+
+1. 'In stock' - total quantity of product items allowed in stock;
+1. 'Reserved'- in VC this field has no logics and is simply a UI element;
```
Comment (mvktsk) [inline README.md]: has no logic?

File: README.md [hunk 3]
```diff
@@ -1,24 +1,70 @@
...
+1. 'Reoder min. qty'- the minimum quantity of products that can be reordered;
+1. 'Allow preoder' - if switched on means that the system will allow order products that are still not in stock. By using the calender, the admin can set up the time period for preoder availability;
```
Comment (mvktsk) [inline README.md]: Typo **preorder** **calendar**

File: README.md [hunk 4]
```diff
@@ -1,24 +1,70 @@
...
+1. 'Preorder qty field'- the admin can specify the product quantity that can be preordered;  
+1. 'Allow backorder'- if switched on means that the system will allow order products that are no longer in stock. The admin can set up the time period for backorder availability;
+1. 'Backorder qty' field- the admin can specify the product quantity that can be backordered.
+
+Some products may not require inventory tracking. For example, services, second hand or custom-made goods. In this case, a store owner can select not to track inventory, by switching off the 'Track Inventory' option on the Physical item details blade
+
+![Track Inventory](docs/media/screen-track-inventory.png)
```
Comment (mvktsk) [inline README.md]: wrong link?

File: README.md [hunk 5]
```diff
@@ -1,24 +1,70 @@
...
+1. Inventory assigned-
+When shipment is created, it is checked if the inventory is available. Afterwards the inventory is being assigned.
+1. Released for Pick/Pack-
+Assigned inventory is released for pick or pack by a customer service representative (CSR). Then the inventory is being picked to the picklist and prepared to be packed
+1. Packing -
+On this step the shipment is packed according to shipping requirements (for instance, fragile items must be packed more accurately and the pack should be marked).
+1. Shipped-
+When the packed shipment is sent to delivery service to be shipped.
+
+[Manage Fulfillment Centers](/docs/manage-fullfilment.md)
```
Comment (mvktsk) [inline README.md]: wrong link?

File: README.md [hunk 6]
```diff
@@ -1,24 +1,70 @@
...
+1. 'Allow backorder'- if switched on means that the system will allow order products that are no longer in stock. The admin can set up the time period for backorder availability;
+1. 'Backorder qty' field- the admin can specify the product quantity that can be backordered.
+
+Some products may not require inventory tracking. For example, services, second hand or custom-made goods. In this case, a store owner can select not to track inventory, by switching off the 'Track Inventory' option on the Physical item details blade
+
+![Track Inventory](docs/media/screen-track-inventory.png)
+
+### Fulfillment Centers
+
+VC fulfillment includes the steps involved in receiving, processing and delivering orders to end customers. Usually the process of fulfillment in ecommerce inlcudes receiving inventory from a supplier (including its counting, instecting, labeling and etc.), storage of the inventory, order processing (including preparing, packing and exchange if necessary) and shipping. There also may be returns that are counted according to the store rules and conditions. In Virto Commerce manager fulfillment lifecycle looks as follows:
```
Comment (mvktsk) [inline README.md]: Typo
Usually **,** the process of fulfillment in ecommerce includes receiving inventory from a supplier (including its counting, **inspecting,** labeling and etc.), storage of the inventory, order processing (including preparing, packing and exchange if necessary) and shipping.

File: README.md [hunk 7]
```diff
@@ -1,24 +1,70 @@
...
+* Manually: download module zip package from https://github.com/VirtoCommerce/vc-module-inventory/releases.
+ In VC Manager go to More -> Modules -> Advanced -> upload module package -> Install.
+
+### Available resources
 
-# Available resources
 * Module related service implementations as a <a href="https://www.nuget.org/packages/VirtoCommerce.InventoryModule.Data" target="_blank">NuGet package</a>
 * API client as a <a href="https://www.nuget.org/packages/VirtoCommerce.InventoryModule.Client" target="_blank">NuGet package</a>
-* API client documentation http://demo.virtocommerce.com/admin/docs/ui/index#!/Inventory_module
+* API client documentation https://admin-demo.virtocommerce.com/#/Inventory%2520module
```
Comment (mvktsk) [inline README.md]: wrong link

File: docs/manage-fullfilment.md [hunk 8]
```diff
@@ -0,0 +1,35 @@
+# Manage Fulfillment Centers
+
+VC Inventroy Module allows managing multiple fulfillment centers. In order to access the list of fulfillment centers, the user should select More->Catalog-> Categories & Items-> Item details-> Inventory widget.
```
Comment (mvktsk) [inline docs/manage-fullfilment.md]: Typo 
**VC Inventory**

File: docs/manage-fullfilment.md [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+
+![Fulfillment Centers](media/screen-fulfillment-centers-list.png)
```
Comment (mvktsk) [inline docs/manage-fullfilment.md]: wrong link ?

File: docs/manage-fullfilment.md [hunk 10]
```diff
@@ -0,0 +1,35 @@
...
+
+## View And Edit Fulfillment Center Details
+
+In order to view the details of a specific fulfillment center and edit it, the user should click the 'Manage Fulfillment Centers' button as seen on the screenshot bellow:
```
Comment (mvktsk) [inline docs/manage-fullfilment.md]: Typo **below**

File: docs/manage-fullfilment.md [hunk 11]
```diff
@@ -0,0 +1,35 @@
...
+1. Name;
+1. Description;
+1. Geo Location;
+1. Address.
+
+Once the details are edited, the user should click the 'Save' button and the changes will be saved.
+
+## Delete Fulfillment Center
+
+The system allows delete the fulfillment centers from the list. In order to delete the selected fulfillment center, the user should select the 'Delete' option under the three dots menu. The system will display a warning message that the fulfillment center will be deleted. The user should confirm the deletion and the fulfillment center will be removed from the system.
```
Comment (mvktsk) [inline docs/manage-fullfilment.md]: allows deleting?

File: docs/manage-fullfilment.md [hunk 12]
```diff
@@ -0,0 +1,35 @@
...
+
+VC Inventroy Module allows managing multiple fulfillment centers. In order to access the list of fulfillment centers, the user should select More->Catalog-> Categories & Items-> Item details-> Inventory widget.
+
+![Fulfillment Centers](media/screen-fulfillment-centers-list.png)
+
+## View And Edit Fulfillment Center Details
+
+In order to view the details of a specific fulfillment center and edit it, the user should click the 'Manage Fulfillment Centers' button as seen on the screenshot bellow:
+
+![Manage Fulfillment](media/screen-manage-centers-button.png)
```
Comment (mvktsk) [inline docs/manage-fullfilment.md]: Wrong link?

File: docs/manage-fullfilment.md [hunk 13]
```diff
@@ -0,0 +1,35 @@
...
+
+The system will open the 'Manage Fulfillment centers' blade. The user should click on the three dots menu located on the left side of the selected fulfillment center and select the 'Manage' option:
+
+![Manage Centers](media/screen-manage-centers.png)
```
Comment (mvktsk) [inline docs/manage-fullfilment.md]: Wrong link?

File: docs/manage-fullfilment.md [hunk 14]
```diff
@@ -0,0 +1,35 @@
...
+
+## Delete Fulfillment Center
+
+The system allows delete the fulfillment centers from the list. In order to delete the selected fulfillment center, the user should select the 'Delete' option under the three dots menu. The system will display a warning message that the fulfillment center will be deleted. The user should confirm the deletion and the fulfillment center will be removed from the system.
+
+## Add Fulfillment Center
+
+The user can add as many fulfillment centers as necessary. To add a new fulfillment center the user should click the 'Add' button on the 'Manage Fulfillment centers' blade and complete the form on 'Edit Fulfillment center' blade. Once the form is completed, the user should save the changes. The new fulfillment center will be added and displayed on the list.
+
+![Add Fulfillment Center](media/screen-add-fulfillment-center.png)
```
Comment (mvktsk) [inline docs/manage-fullfilment.md]: Wrong link?

File: README.md (hunk 2 above)
Comment (mayaalsamarrai) [reply README.md]: Logics! Yes, means it's simply a UI element. May be I should rephrase it ?

File: README.md (hunk 4 above)
Comment (mayaalsamarrai) [reply README.md]: This screenshot is displayed correctly on my side

File: README.md (hunk 5 above)
Comment (mayaalsamarrai) [reply README.md]: It works correctly on my side

File: README.md (hunk 7 above)
Comment (mayaalsamarrai) [reply README.md]: Yes, this one is wrong

File: docs/manage-fullfilment.md (hunk 9 above)
Comment (mayaalsamarrai) [reply docs/manage-fullfilment.md]: Displayed correctly on my side

File: docs/manage-fullfilment.md (hunk 12 above)
Comment (mayaalsamarrai) [reply docs/manage-fullfilment.md]: displayed correctly on my side

File: docs/manage-fullfilment.md (hunk 13 above)
Comment (mayaalsamarrai) [reply docs/manage-fullfilment.md]: The same- correct on my side

File: docs/manage-fullfilment.md (hunk 14 above)
Comment (mayaalsamarrai) [reply docs/manage-fullfilment.md]: The same! No wrong links on my side, lets investigate the issue

File: README.md (hunk 2 above)
Comment (mvktsk) [reply README.md]: Rephrase it please

File: README.md [hunk 15]
```diff
@@ -1,24 +1,72 @@
...
+The Inventory module content can be both exported and imported using the VC Export/Import functionality.
+
+![Inventory module](docs/media/screen-inventory.png)
+
+Fields description :
+
+1. 'In stock' - total quantity of product items allowed in stock;
+1. 'Reserved'- in VC this field has no logics and is simply a UI element;
+1. 'Reoder min. qty'- the minimum quantity of products that can be reordered;
+1. 'Allow preoder' - if switched on means that the system will allow order products that are still not in stock. By using the preorder calender, the admin can set up the time period for preoder availability;
```
Comment (mvktsk) [inline README.md]: Typo **preorder calendar**

**Comment by mayaalsamarrai** [comment]
Edited based on the review

**Pull request merged**